admin/post-types/writepanels/writepanel-course_data.php

```diff
@@ -68,38 +70,38 @@ function course_certificate_template_data_meta_box( $post ) {
 		$html .= '<option value="">' . __( 'None', 'sensei-certificates' ) . '</option>';
 		foreach ( $posts_array as $post_item ) {
 			$html .= '<option value="' . esc_attr( absint( $post_item->ID ) ) . '"' . selected( $post_item->ID, $select_certificate_template, false ) . '>' . esc_html( $post_item->post_title ) . '</option>' . "\n";
-		} // End For Loop
+		}
 		$html .= '</select>' . "\n";
 	} else {
 		if ( ! empty( $select_certificate_template ) ) {
 			$html .= '<input type="hidden" name="course_certificate_template" value="' . absint( $select_certificate_template ) . '">';
 		}
 		$html .= '<p>' . esc_html( __( 'No certificate template exist yet. Please add some first.', 'sensei-certificates' ) ) . '</p>';
-	} // End If Statement
+	}
 
 	echo $html;
 
-} // End course_certificate_templates_data_meta_box()
+}
 
 
 /**
- * Course Certificate Template Data Save
+ * Course Certificate Template Data Save.
  *
  * Function for processing and storing all course certificate data.
  *
  * @since 1.0.0
- * @param int    $post_id the certificate id
- * @param object $post the certificate post object
+ * @param int    $post_id The certificate id.
+ * @param object $post    The certificate post object.
  */
 function course_certificate_templates_process_meta( $post_id ) {
 
 	global $woothemes_sensei_certificate_templates;
 
-	/* Verify the nonce before proceeding. */
+	// Verify the nonce before proceeding.
```

Comment:
Old comment is a bit misleading. We can probably just remove it. Nonce is checked inside `save_post_meta`

Reply:
Fixed here: https://github.com/woocommerce/sensei-certificates/pull/219/commits/7fefb5d8535bf53f184b9d9b8015fd23c335d79d